dark-mode.css

```diff
@@ -139,12 +175,24 @@ body.dark-mode .sidebar-item > a {
 	color: inherit;
 }
 
-body.dark-mode .emoji-picker {
-	background-color: var(--rc-color-primary-dark);
+body.dark-mode input {
```

Comment:
I think you should delete this element. It ends up messing up either the admin side or the front side. Removing these three lines entirely allows them to nicely default the the appropriate colors

Reply:
I checked that deletion of 
```
body.dark-mode input {
	color: var(--color-white);
}
```
solved my problem in Admin panel

Reply:
Agreed - not sure why this rule existed to begin with, so I'm removing.